tests/conftest.py

```diff
@@ -116,7 +111,7 @@ def openai_api_key(monkeypatch: pytest.MonkeyPatch) -> str:
 postgresql_connection = factories.postgresql("postgresql_proc")
 
 
-@pytest.fixture()
+@pytest.fixture(scope="function")
```

Comment:
Isn't "function" the default scope?

Reply:
yes, but it's better to be explicit in case the default changes

Reply:
okay. fwiw i doubt pytest will change that default since it would be a very disruptive change.